src/Microsoft.Dnx.Tooling/Utils/PackageSourceUtils.cs

```diff
@@ -14,6 +15,8 @@ namespace Microsoft.Dnx.Tooling
 {
     public static class PackageSourceUtils
     {
+        private static ConcurrentDictionary<string, HttpSource> _httpSources = new ConcurrentDictionary<string, HttpSource>();
```

Comment:
Could we pass in the cache instead? Hard to know what the lifetime of these `HttpSource` instances are with statics and if it's appropriate to cache them indefinitely.